slick.grid.js

```diff
@@ -347,14 +348,21 @@ if (typeof Slick === "undefined") {
       // ----------------------- Matryoshka the elements together
       topCanvas.el[0].appendChild(header.el[0]);
       topCanvas.el[1].appendChild(header.el[1]);
-      topCanvas.el[0].appendChild(subHeaders.el[0]);
-      topCanvas.el[1].appendChild(subHeaders.el[1]);
       topViewport.el[0].appendChild(topCanvas.el[0]);
       topViewport.el[1].appendChild(topCanvas.el[1]);
       contentViewport.el[0].appendChild(contentCanvas.el[0]);
       contentViewport.el[1].appendChild(contentCanvas.el[1]);
       contentViewportWrap.el[0].appendChild(contentViewport.el[0]);
       contentViewportWrap.el[1].appendChild(contentViewport.el[1]);
+
+      // TODO: support subHeaders.el[1]

```

Comment:
for now, no. if we ahve a use case in the future, we'll have to inject the subheaders into a container  with left/right subcontainers and size them appropriately, just like what we already do for the header and body. punting for now.